README.md

```diff
@@ -427,15 +461,6 @@ applications.
 To run these steps automatically you may include the additional package
 `bootleg_phoenix` in your `deps` list. This package provides the build hook commands required to build most Phoenix releases.
 
-```elixir
-# mix.exs
-def deps do
-  [{:distillery, "~> 1.5"},
-  {:bootleg, "~> 0.6"},
-  {:bootleg_phoenix, "~> 0.2"}]
-end
-```
-
```

Comment:
We don't need to include sample deps for additional packages, when doing so it just makes it harder to update versions

Reply:
👍